sql/src/main/java/io/crate/metadata/settings/CrateTableSettings.java

```diff
@@ -50,6 +50,9 @@
         "all"
     );
 
+    public static final StringSetting SETTING_WAIT_FOR_ACTIVE_SHARDS= new StringSetting(
+        TableParameterInfo.SETTING_WAIT_FOR_ACTIVE_SHARDS, null, "all");
```

Comment:
Please use the default value from the actual setting definition to prevent this from getting out of sync.